markdown/postprocessors.py

```diff
@@ -75,9 +75,21 @@ def run(self, text):
                     self.md.htmlStash.get_placeholder(i))] = html
             replacements[self.md.htmlStash.get_placeholder(i)] = html
 
+        def substitute_match(m):
+            key = m.group(0)
+
+            if key not in replacements:
+                if key[3:-4] in replacements:
+                    return f'<p>{ replacements[key[3:-4]] }</p>'
+                else:
```

Comment:
This line will never be reached as per [this comment](https://github.com/Python-Markdown/markdown/pull/1083#discussion_r538580902)

Reply:
Except that it is in the test I added with this fix (see the [coverage report][1]). The assumption is that a third-party extension could create this scenario. We never want to allow a situation where the parser could raise an error based on user input. Therefore, we are leaving this in.

[1]: https://github.com/Python-Markdown/markdown/runs/1518569326

Reply:
I see. I had not delved into the interaction of extensions with this code. Thank you for your help and for accepting the overall changes.